src/seth/libexec/seth/seth-abi-encode

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+### seth-abi-encode -- ABI encode values, and prints the encoded values without the function signature
+### Usage: seth abi-encode <sig> [<args>]
```

Comment:
Does it make sense to accept a tuple of types and then automatically add some dummy function signature just as an internal implementation detail?

Reply:
You could, but IMO we should keep the current `sig` input format since it's more familiar (i.e. used in other seth methods) and as a result more composable with other methods (i.e. outputs from the 4byte methods).

Also noticed a docs issue with this command, so fixed in b8f25aff1c11eb8022a83a633121373cf48c8652